docs/Contributing-to-the-Wiki.md

```diff
@@ -0,0 +1,11 @@
+### How is the Wiki Updated?
```

Comment:
You can remove this file since the information is already present in the readme.
OR
Keep the readme and let me point to the Home page. 

Keep either one instead of duplicating. 

Thanks!

Reply:
Thanks for the quick reply!

My understanding of your suggestions is:

Option 1: Keep only the readme file. The readme file will not be visible on the wiki as a result of this

Option 2: Add the readme file to the wiki and the wiki's home page. The readme file must contain absolute URLs though otherwise linkchecker fails. 

I am personally leaning towards option 2.

Reply:
Please go-ahead with option 2. Link checker fail can be fixed later after by-passing it once for initial commit.
Thanks!